<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<script src="libraries/htmlwidgets-1.3/htmlwidgets.js"></script>
<script src="libraries/plotly-binding-4.9.0/plotly.js"></script>
<script src="libraries/pymjs-1.3.2/pym.v1.min.js"></script>
<script src="libraries/typedarray-0.1/typedarray.min.js"></script>
<script src="libraries/jquery-1.11.3/jquery.min.js"></script>
<link href="libraries/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libraries/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libraries/plotly-htmlwidgets-css-1.46.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libraries/plotly-main-1.46.1/plotly-latest.min.js"></script>
  <title>plotly</title>
</head>
<body style="background-color:white;">
<div id="htmlwidget_container">
  <div id="htmlwidget-dbdce703bc4bb94e5782" style="width:100%;height:400px;" class="plotly html-widget"></div>
  <script>HTMLWidgets.pymChild = new pym.Child();HTMLWidgets.addPostRenderHandler(function(){
                                setTimeout(function(){HTMLWidgets.pymChild.sendHeight();},100);
                            });</script>
</div>
<script type="application/json" data-for="htmlwidget-dbdce703bc4bb94e5782">{"x":{"data":[{"orientation":"v","width":[0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":["Tested On Non Human Data","Visualisation","Error Correction And Polishing","Denovo Assembly","SNP And Variant Analysis","Quality Checking","Availablity Of Test Data","Analysis Pipelines","Provide Summary Statistics","Alignment","Tested On Human Data","Base Calling","Generating Consensus Sequence","Base Modification Detection","Metagenomics","Taxonomic Classification","Isoform Detection","Simulators","Gene Expression Analysis","Quality Trimming","Evaluating Exisiting Methods","Quality Filtering","Long Read Overlapping","fast5File Processing","Read Quantification","Scaffolding","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling","Normalisation","poly ALength Estimation"],"y":[0.151315789473684,0.118421052631579,0.115131578947368,0.111842105263158,0.108552631578947,0.0986842105263158,0.0921052631578947,0.0888157894736842,0.0888157894736842,0.0822368421052632,0.0789473684210526,0.0690789473684211,0.0592105263157895,0.0493421052631579,0.0493421052631579,0.0460526315789474,0.0394736842105263,0.0394736842105263,0.0296052631578947,0.0296052631578947,0.0263157894736842,0.0263157894736842,0.0230263157894737,0.0197368421052632,0.0131578947368421,0.0131578947368421,0.00986842105263158,0.00657894736842105,0.00657894736842105,0.00657894736842105,0.00657894736842105],"text":["count: 0.151315789<br />Category: Tested On Non Human Data<br />Prop: 0.151315789<br />Count: 46 <br /> Percent: 15.1","count: 0.118421053<br />Category: Visualisation<br />Prop: 0.118421053<br />Count: 36 <br /> Percent: 11.8","count: 0.115131579<br />Category: Error Correction And Polishing<br />Prop: 0.115131579<br />Count: 35 <br /> Percent: 11.5","count: 0.111842105<br />Category: Denovo Assembly<br />Prop: 0.111842105<br />Count: 34 <br /> Percent: 11.2","count: 0.108552632<br />Category: SNP And Variant Analysis<br />Prop: 0.108552632<br />Count: 33 <br /> Percent: 10.9","count: 0.098684211<br />Category: Quality Checking<br />Prop: 0.098684211<br />Count: 30 <br /> Percent: 9.9","count: 0.092105263<br />Category: Availablity Of Test Data<br />Prop: 0.092105263<br />Count: 28 <br /> Percent: 9.2","count: 0.088815789<br />Category: Analysis Pipelines<br />Prop: 0.088815789<br />Count: 27 <br /> Percent: 8.9","count: 0.088815789<br />Category: Provide Summary Statistics<br />Prop: 0.088815789<br />Count: 27 <br /> Percent: 8.9","count: 0.082236842<br />Category: Alignment<br />Prop: 0.082236842<br />Count: 25 <br /> Percent: 8.2","count: 0.078947368<br />Category: Tested On Human Data<br />Prop: 0.078947368<br />Count: 24 <br /> Percent: 7.9","count: 0.069078947<br />Category: Base Calling<br />Prop: 0.069078947<br />Count: 21 <br /> Percent: 6.9","count: 0.059210526<br />Category: Generating Consensus Sequence<br />Prop: 0.059210526<br />Count: 18 <br /> Percent: 5.9","count: 0.049342105<br />Category: Base Modification Detection<br />Prop: 0.049342105<br />Count: 15 <br /> Percent: 4.9","count: 0.049342105<br />Category: Metagenomics<br />Prop: 0.049342105<br />Count: 15 <br /> Percent: 4.9","count: 0.046052632<br />Category: Taxonomic Classification<br />Prop: 0.046052632<br />Count: 14 <br /> Percent: 4.6","count: 0.039473684<br />Category: Isoform Detection<br />Prop: 0.039473684<br />Count: 12 <br /> Percent: 3.9","count: 0.039473684<br />Category: Simulators<br />Prop: 0.039473684<br />Count: 12 <br /> Percent: 3.9","count: 0.029605263<br />Category: Gene Expression Analysis<br />Prop: 0.029605263<br />Count: 9 <br /> Percent: 3","count: 0.029605263<br />Category: Quality Trimming<br />Prop: 0.029605263<br />Count: 9 <br /> Percent: 3","count: 0.026315789<br />Category: Evaluating Exisiting Methods<br />Prop: 0.026315789<br />Count: 8 <br /> Percent: 2.6","count: 0.026315789<br />Category: Quality Filtering<br />Prop: 0.026315789<br />Count: 8 <br /> Percent: 2.6","count: 0.023026316<br />Category: Long Read Overlapping<br />Prop: 0.023026316<br />Count: 7 <br /> Percent: 2.3","count: 0.019736842<br />Category: fast5File Processing<br />Prop: 0.019736842<br />Count: 6 <br /> Percent: 2","count: 0.013157895<br />Category: Read Quantification<br />Prop: 0.013157895<br />Count: 4 <br /> Percent: 1.3","count: 0.013157895<br />Category: Scaffolding<br />Prop: 0.013157895<br />Count: 4 <br /> Percent: 1.3","count: 0.009868421<br />Category: Suitable For Single Cell Experiments<br />Prop: 0.009868421<br />Count: 3 <br /> Percent: 1","count: 0.006578947<br />Category: Demultiplexing<br />Prop: 0.006578947<br />Count: 2 <br /> Percent: 0.7","count: 0.006578947<br />Category: Gap Filling<br />Prop: 0.006578947<br />Count: 2 <br /> Percent: 0.7","count: 0.006578947<br />Category: Normalisation<br />Prop: 0.006578947<br />Count: 2 <br /> Percent: 0.7","count: 0.006578947<br />Category: poly ALength Estimation<br />Prop: 0.006578947<br />Count: 2 <br /> Percent: 0.7"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(159,199,81,1)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":50,"r":40,"b":90,"l":70},"font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693},"title":{"text":"<b> Oxford Nanopore focused <\/b>","font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"x":0.5,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"category","autorange":true,"range":[0.4,31.6],"tickmode":"auto","ticktext":["Tested On Non Human Data","Visualisation","Error Correction And Polishing","Denovo Assembly","SNP And Variant Analysis","Quality Checking","Availablity Of Test Data","Analysis Pipelines","Provide Summary Statistics","Alignment","Tested On Human Data","Base Calling","Generating Consensus Sequence","Base Modification Detection","Metagenomics","Taxonomic Classification","Isoform Detection","Simulators","Gene Expression Analysis","Quality Trimming","Evaluating Exisiting Methods","Quality Filtering","Long Read Overlapping","fast5File Processing","Read Quantification","Scaffolding","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling","Normalisation","poly ALength Estimation"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31],"categoryorder":"array","categoryarray":["Tested On Non Human Data","Visualisation","Error Correction And Polishing","Denovo Assembly","SNP And Variant Analysis","Quality Checking","Availablity Of Test Data","Analysis Pipelines","Provide Summary Statistics","Alignment","Tested On Human Data","Base Calling","Generating Consensus Sequence","Base Modification Detection","Metagenomics","Taxonomic Classification","Isoform Detection","Simulators","Gene Expression Analysis","Quality Trimming","Evaluating Exisiting Methods","Quality Filtering","Long Read Overlapping","fast5File Processing","Read Quantification","Scaffolding","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling","Normalisation","poly ALength Estimation"],"nticks":null,"ticks":"outside","tickcolor":"rgba(0,0,0,1)","ticklen":4.64923204649232,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(0,0,0,1)","family":"","size":11.9551681195517},"tickangle":-60,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":true,"range":[-0.00756578947368421,0.158881578947368],"tickmode":"auto","ticktext":["0.0%","5.0%","10.0%","15.0%"],"tickvals":[0,0.05,0.1,0.15],"categoryorder":"array","categoryarray":["0.0%","5.0%","10.0%","15.0%"],"nticks":null,"ticks":"outside","tickcolor":"rgba(0,0,0,1)","ticklen":4.64923204649232,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(0,0,0,1)","family":"","size":11.9551681195517},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":{"text":"Percentage of tools","font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"3db8d2e5a42":{"x":{},"weight":{},"text":{},"type":"bar"}},"cur_data":"3db8d2e5a42","visdat":{"3db8d2e5a42":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-dbdce703bc4bb94e5782">{"viewer":{"width":"100%","height":400,"padding":0,"fill":true},"browser":{"width":"100%","height":400,"padding":0,"fill":true}}</script>
</body>
</html>
