<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<script src="libraries/htmlwidgets-1.3/htmlwidgets.js"></script>
<script src="libraries/plotly-binding-4.8.0/plotly.js"></script>
<script src="libraries/pymjs-1.3.2/pym.v1.min.js"></script>
<script src="libraries/typedarray-0.1/typedarray.min.js"></script>
<script src="libraries/jquery-1.11.3/jquery.min.js"></script>
<link href="libraries/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libraries/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libraries/plotly-htmlwidgets-css-1.39.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libraries/plotly-main-1.39.2/plotly-latest.min.js"></script>
  <title>plotly</title>
</head>
<body style="background-color:white;">
<div id="htmlwidget_container">
  <div id="htmlwidget-f2fa4fe74c8ce391c931" style="width:100%;height:400px;" class="plotly html-widget"></div>
  <script>HTMLWidgets.pymChild = new pym.Child();HTMLWidgets.addPostRenderHandler(function(){
                                setTimeout(function(){HTMLWidgets.pymChild.sendHeight();},100);
                            });</script>
</div>
<script type="application/json" data-for="htmlwidget-f2fa4fe74c8ce391c931">{"x":{"data":[{"orientation":"v","width":[0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":["Tested On Non Human Data","Denovo Assembly","Quality Checking","Visualisation","Availablity Of Test Data","Analysis Pipelines","Error Correction And Polishing","Provide Summary Statistics","SNP And Variant Analysis","Alignment","Base Calling","Generating Consensus Sequence","Tested On Human Data","Base Modification Detection","Metagenomics","Gene Expression Analysis","Isoform Detection","Quality Trimming","Quality Filtering","Long Read Overlapping","Simulators","Evaluating Exisiting Methods","Read Quantification","Normalisation","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling"],"y":[0.159259259259259,0.111111111111111,0.107407407407407,0.107407407407407,0.1,0.0962962962962963,0.0962962962962963,0.0888888888888889,0.0888888888888889,0.0814814814814815,0.0666666666666667,0.0666666666666667,0.0666666666666667,0.0555555555555556,0.0518518518518519,0.037037037037037,0.037037037037037,0.0333333333333333,0.0296296296296296,0.0259259259259259,0.0259259259259259,0.0111111111111111,0.0111111111111111,0.00740740740740741,0.00740740740740741,0.0037037037037037,0],"text":["count: 0.159259259<br />Category: Tested On Non Human Data<br />Prop: 0.159259259<br />Count: 43 <br /> Percent: 15.9","count: 0.111111111<br />Category: Denovo Assembly<br />Prop: 0.111111111<br />Count: 30 <br /> Percent: 11.1","count: 0.107407407<br />Category: Quality Checking<br />Prop: 0.107407407<br />Count: 29 <br /> Percent: 10.7","count: 0.107407407<br />Category: Visualisation<br />Prop: 0.107407407<br />Count: 29 <br /> Percent: 10.7","count: 0.100000000<br />Category: Availablity Of Test Data<br />Prop: 0.100000000<br />Count: 27 <br /> Percent: 10","count: 0.096296296<br />Category: Analysis Pipelines<br />Prop: 0.096296296<br />Count: 26 <br /> Percent: 9.6","count: 0.096296296<br />Category: Error Correction And Polishing<br />Prop: 0.096296296<br />Count: 26 <br /> Percent: 9.6","count: 0.088888889<br />Category: Provide Summary Statistics<br />Prop: 0.088888889<br />Count: 24 <br /> Percent: 8.9","count: 0.088888889<br />Category: SNP And Variant Analysis<br />Prop: 0.088888889<br />Count: 24 <br /> Percent: 8.9","count: 0.081481481<br />Category: Alignment<br />Prop: 0.081481481<br />Count: 22 <br /> Percent: 8.1","count: 0.066666667<br />Category: Base Calling<br />Prop: 0.066666667<br />Count: 18 <br /> Percent: 6.7","count: 0.066666667<br />Category: Generating Consensus Sequence<br />Prop: 0.066666667<br />Count: 18 <br /> Percent: 6.7","count: 0.066666667<br />Category: Tested On Human Data<br />Prop: 0.066666667<br />Count: 18 <br /> Percent: 6.7","count: 0.055555556<br />Category: Base Modification Detection<br />Prop: 0.055555556<br />Count: 15 <br /> Percent: 5.6","count: 0.051851852<br />Category: Metagenomics<br />Prop: 0.051851852<br />Count: 14 <br /> Percent: 5.2","count: 0.037037037<br />Category: Gene Expression Analysis<br />Prop: 0.037037037<br />Count: 10 <br /> Percent: 3.7","count: 0.037037037<br />Category: Isoform Detection<br />Prop: 0.037037037<br />Count: 10 <br /> Percent: 3.7","count: 0.033333333<br />Category: Quality Trimming<br />Prop: 0.033333333<br />Count: 9 <br /> Percent: 3.3","count: 0.029629630<br />Category: Quality Filtering<br />Prop: 0.029629630<br />Count: 8 <br /> Percent: 3","count: 0.025925926<br />Category: Long Read Overlapping<br />Prop: 0.025925926<br />Count: 7 <br /> Percent: 2.6","count: 0.025925926<br />Category: Simulators<br />Prop: 0.025925926<br />Count: 7 <br /> Percent: 2.6","count: 0.011111111<br />Category: Evaluating Exisiting Methods<br />Prop: 0.011111111<br />Count: 3 <br /> Percent: 1.1","count: 0.011111111<br />Category: Read Quantification<br />Prop: 0.011111111<br />Count: 3 <br /> Percent: 1.1","count: 0.007407407<br />Category: Normalisation<br />Prop: 0.007407407<br />Count: 2 <br /> Percent: 0.7","count: 0.007407407<br />Category: Suitable For Single Cell Experiments<br />Prop: 0.007407407<br />Count: 2 <br /> Percent: 0.7","count: 0.003703704<br />Category: Demultiplexing<br />Prop: 0.003703704<br />Count: 1 <br /> Percent: 0.4","count: 0.000000000<br />Category: Gap Filling<br />Prop: 0.000000000<br />Count: 0 <br /> Percent: 0"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(159,199,81,1)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":50,"r":40,"b":90,"l":70},"font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693},"title":"<b> Oxford Nanopore focused <\/b>","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"xaxis":{"domain":[0,1],"automargin":true,"type":"category","autorange":true,"range":[0.4,27.6],"tickmode":"auto","ticktext":["Tested On Non Human Data","Denovo Assembly","Quality Checking","Visualisation","Availablity Of Test Data","Analysis Pipelines","Error Correction And Polishing","Provide Summary Statistics","SNP And Variant Analysis","Alignment","Base Calling","Generating Consensus Sequence","Tested On Human Data","Base Modification Detection","Metagenomics","Gene Expression Analysis","Isoform Detection","Quality Trimming","Quality Filtering","Long Read Overlapping","Simulators","Evaluating Exisiting Methods","Read Quantification","Normalisation","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27],"categoryorder":"array","categoryarray":["Tested On Non Human Data","Denovo Assembly","Quality Checking","Visualisation","Availablity Of Test Data","Analysis Pipelines","Error Correction And Polishing","Provide Summary Statistics","SNP And Variant Analysis","Alignment","Base Calling","Generating Consensus Sequence","Tested On Human Data","Base Modification Detection","Metagenomics","Gene Expression Analysis","Isoform Detection","Quality Trimming","Quality Filtering","Long Read Overlapping","Simulators","Evaluating Exisiting Methods","Read Quantification","Normalisation","Suitable For Single Cell Experiments","Demultiplexing","Gap Filling"],"nticks":null,"ticks":"outside","tickcolor":"rgba(0,0,0,1)","ticklen":4.64923204649232,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(0,0,0,1)","family":"","size":11.9551681195517},"tickangle":-60,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","titlefont":{"color":null,"family":null,"size":0},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":true,"range":[-0.00796296296296296,0.167222222222222],"tickmode":"auto","ticktext":["0.0%","5.0%","10.0%","15.0%"],"tickvals":[0,0.05,0.1,0.15],"categoryorder":"array","categoryarray":["0.0%","5.0%","10.0%","15.0%"],"nticks":null,"ticks":"outside","tickcolor":"rgba(0,0,0,1)","ticklen":4.64923204649232,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(0,0,0,1)","family":"","size":11.9551681195517},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Percentage of tools","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":null,"bordercolor":null,"borderwidth":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"source":"A","attrs":{"173412c815a5":{"x":{},"weight":{},"text":{},"type":"bar"}},"cur_data":"173412c815a5","visdat":{"173412c815a5":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-f2fa4fe74c8ce391c931">{"viewer":{"width":"100%","height":400,"padding":0,"fill":true},"browser":{"width":"100%","height":400,"padding":0,"fill":true}}</script>
</body>
</html>
